docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
+diverse team of community members, documentarians, and core developers.
+
+Things the workgroup could do:
+
+- Develop the governance model for docs
+- Be the steering council for docs
+- Be the authors of doc guidelines
+- Be the authors/editors of docs
+- Request grant funding from PSF, if needed
+- Form an editorial board for docs including core devs, educators, and
+  documentarians in the Python community
+
+Active Time
+-----------
+
+This workgroup will be active indefinitely while supported by the Python
+Steering Council.
+
+Core Values & Internal Governance
+---------------------------------
+
+The core values for this group are:
+
+- Respect
+- Support
+- Positive Atmosphere
+- Constructiveness
+- Integrity
+
+The workgroup adopts the `PSF Code of Conduct <https://www.python.org/psf/codeofconduct/>`_.
+Any action by a workgroup member, as decided by a majority of the group,
+that violates the principles in the Code of Conduct will result in that member
+being removed from the workgroup.
```

Comment:
Pedantic governance point -- how does this fit with "*The Python Steering Council are permanent members of this working group*"? Are such ex-officio members included in this section?

Reply:
The CoC governs SC members just like any other member of the PSF/Python project. If they committed a suitably serious CoC violation sufficient to merit removal, presumably that would be handled by the CoC committee rather than us, but they would be removed from the SC, and thus from the workgroup as well.

Reply:
I agree. That's not what this text says though -- and whilst unlikely we should be accurate. I'm happy to suggest this one (and the TODO one above) in a follow-up PR if you don't want to include here.

A

Reply:
IMO, this is a bit too pedantic to need to be explicitly denoted in the work(ing)? ?group's founding charter, but its ultimately up to the work group members, not either of us.